### DISCUSSION TITLE
1.3.0

### DISCUSSION THREAD

**Discussion by jglev** [jglev/obsidian-carry-forward#8] [Announcements]
- Restructured and re-wrote Settings explanatory text (following #6)
- Added commands for using selection or current clipboard text as copied link text (#5; thanks to @FelipeRearden for writing up the use-case)
- Fixed links in Readme (also thanks to @FelipeRearden in #2)

Due to a clerical error, version `1.2.0` got skipped -- so we went straight from `1.1.0` to `1.3.0` :smile: 

<hr /><em>This discussion was created from the release <a href='https://github.com/jglev/obsidian-carry-forward/releases/tag/1.3.0'>1.3.0</a>.</em>

**Comment by FelipeRearden** [comment]
Working prefect on version `1.3.0` Obsidian `0.13.19` !!!!!!
- Source Mode
- Live Preview

All 12 commands were tested 

The new UI is great !!!!!!!

Thank you very much @jglev !!!!!!!!!!!!!!!!!!!!!!!!

**Reply by jglev** [reply]
Thank you for letting me know! : )
